network/p2p/handler.go

```diff
@@ -21,6 +21,11 @@ var (
 	_ Handler = (*ValidatorHandler)(nil)
 )
 
+const (
+	TxGossipHandlerID = iota
+	SignatureRequestHandlerID
```

Comment:
This actually conflicts with the existing `atomicTxGossipHandlerID` used by `coreth`. We should probably update this and the ACP accordingly.

Reply:
Bumped to `2` by adding `AtomicTxGossipHandlerID` here. Corresponding ACP change: https://github.com/avalanche-foundation/ACPs/pull/132